extra/nrnivmodl_core_makefile.in

```diff
@@ -241,3 +241,6 @@ install: $(SPECIAL_EXE) coremech_lib_target
 .PHONY: build_always
 
 $(VERBOSE).SILENT:
+
+# delete cpp files if mod2c error, otherwise they are not generated again
+.DELETE_ON_ERROR:
```

Comment:
```
.DELETE_ON_ERROR
If .DELETE_ON_ERROR is mentioned as a target anywhere in the makefile, then make will delete the target of a rule if it has changed and its recipe exits with a nonzero exit status, just as it does when it receives a signal. 
```

Never used this before! :star: Does this work as expected?

Reply:
Yes ;) Works like a charm.